app/components/GroupForm/index.js

```diff
@@ -67,7 +67,7 @@ function GroupForm({
         label="Beskrivelse"
         placeholder="Vil du strikke din egen lue? Eller har du allerede […]"
         name="text"
-        component={EditorField.Field}
+        component={EditorField}
```

Comment:
Does this work? :thinking:

Reply:
It didn't work with the field, and this works fine. Dont know why :shrug:

Reply:
Ahh. Will however break a lot of the redux-form features

Reply:
Yeah, I will have to take a look at it, had some weird bugs while using the field. Might have been fixed by the async onFocus and onBlur props though.